src/Cms/App.php

```diff
@@ -529,6 +529,43 @@ public function core()
         return $this->core;
     }
 
+    /**
+     * Checks/returns a CSRF token
+     * @since 3.7.0
+     *
+     * @param string|null $check Pass a token here to compare it to the one in the session
+     * @return string|bool Either the token or a boolean check result
+     */
+    public static function csrf(?string $check = null)
```

Comment:
We could make this an instance method to be in line with `session()` and all the others. Then we wouldn't need to use `static::instance()` inside but just `$this`.